Misc/NEWS.d/next/Library/2019-02-10-16-49-16.bpo-21269.Fqi7VH.rst

```diff
@@ -0,0 +1 @@
+Add args and kwards propery to mock call objects. Contributed by Kumar Akshay.
```

Comment:
Typos in `kwards` and `propery` need fixing. args and kwargs should likely be surrounded by double back ticks, as in the docs, as it looks line NEWS is ReST too.